assesspy/metrics.py

```diff
@@ -0,0 +1,375 @@
+import math
+from typing import Union
+
+import pandas as pd
+import statsmodels.api as sm
+
+from .utils import check_inputs
+
+
+def cod(
+    estimate: Union[list[int], list[float], pd.Series],
```

Comment:
Using `Union` here instead of the newer `list[int] | list[float]` format for compatibility with Python 3.9.